src/App.tsx

```diff
@@ -1,14 +1,54 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { Status } from './types/Status';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { FilterTodo } from './service/FilterTodo';
+import { getTodos } from './service/todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedValue, setSelectedValue] = useState(Status.All);
+  const [isLoading, setIsLoading] = useState(true);
+  const [text, setText] = useState('');
+  const [selectedtodo, setSelectedTodo] = useState<Todo | undefined>();
```

Comment:
it's better to use like this
```suggestion
  const [selectedtodo, setSelectedTodo] = useState<Todo | null>(null);
```